### DISCUSSION TITLE
Pod-to-Pod Multi-cluster communication error

### DISCUSSION THREAD

**Discussion by polushynd** [linkerd/linkerd2#11720] [General]
Hi everyone, I am trying to use: Pod-to-Pod Multi-cluster communication ([https://linkerd.io/2.14/tasks/pod-to-pod-multicluster/?_gl=1*1b70s5h*_ga*MTYzNjYwN[…]zU2*_ga_TV358ZPK6D*MTcwMDgzNzg5NC4zOS4xLjE3MDA4Mzc5MjEuMzMuMC4w](https://linkerd.io/2.14/tasks/pod-to-pod-multicluster/?_gl=1*1b70s5h*_ga*MTYzNjYwNjEwNy4xNjkyMDM1NzU2*_ga_TV358ZPK6D*MTcwMDgzNzg5NC4zOS4xLjE3MDA4Mzc5MjEuMzMuMC4w#)), 2 eks clusters (peering connect), 

So I installed everything and trying to test the connection, but got the following error:

> curl http://bb-faas-dev-cluster:8080/ -i
HTTP/1.1 504 Gateway Timeout
l5d-proxy-error: logical service 172.20.247.14:8080: route default.http: backend default.service: service in fail-fast
connection: close
l5d-proxy-connection: close
content-length: 0
date: Fri, 08 Dec 2023 16:25:11 GMT

and linkerd check returns:

> linkerd-multicluster
× probe services able to communicate with all gateway mirrors
        wrong number (0) of probe gateways for target cluster faas-dev-cluster
    see https://linkerd.io/2.14/checks/#l5d-multicluster-gateways-endpoints for hints

But according to the doc: This is exactly the same as in the regular [Multicluster guide](https://linkerd.io/2.14/tasks/multicluster/#linking-the-clusters) except that we pass the --gateway=false flag to create a Link which doesn’t require a gateway.

**Comment by alpeb** [comment]
Hi, I think the `linkerd check` error can be disregarded as in this case because, as you pointed out, there's no gateway to probe. There's not much information here to diagnose the problem. We can start by looking at the proxy logs on the target service.
